src/support/steps.js

```diff
@@ -60,19 +60,23 @@ module.exports = function () {
 	});
 
 	this.Then(/^the element "([^"]*)" will have the class "([^"]*)"$/, function (semanticName, className, callback) {
-		this.client.waitForExist(this.component[semanticName], this.defaultTimeout).then(function () {
-			return this.client.getAttribute(this.component[semanticName], 'class');
-		}.bind(this))
-			.then(function (attr) {
-				try {
-					var classList = helper.flattenClassList(attr);
-					this.assert.notEqual(_.indexOf(classList, className), -1);
-					callback();
-				} catch (e) {
-					callback(e);
-				}
+		if (className === '') {
```

Comment:
One of the use cases has blank class and I would like to include it in Scenario Outline examples, eg:
		Examples:
		| usecase			| class	| text 			|
		| info				| info		| Info			|
		| regular			| 		| Default			|